src/main/java/org/example/Person.java

```diff
@@ -0,0 +1,108 @@
+package org.example;
+
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * This is a class with 9 methods
+ */
+
+public class Person {
+    public String name;
+    public int age;
+    public String type;
+
+    public List<String> listOfHobbies;
+
+    /**
+     * This is the constructor for the class Person
+     * @param name;
+     * @param age;
+     * @param type;
+     */
+
+    public Person(String name, int age, String type) {
+        this.name = name;
+        this.age = age;
+        this.type = type;
+        this.listOfHobbies = new ArrayList<String>();
+    }
+
+    /**
+     * This is a method that adds the hobby to the listOfHobbies
+     * @param hobby;
+     */
+    public void addHobby(String hobby) {
```

Comment:
get rid of this (and the list) because you have the Map of Person and hobbies in main